bufstream/quickstart/docker-compose.yml

```diff
@@ -0,0 +1,152 @@
+      start_period: 15s
+    volumes:
+      - pgData:/var/lib/postgresql/data
+
+  minio:
+    image: minio/minio
+    container_name: minio
+    environment:
+      - MINIO_ROOT_USER=admin
+      - MINIO_ROOT_PASSWORD=password
+      - MINIO_DOMAIN=minio
+    networks:
+      bufstream:
+        aliases:
+          - bufstream.minio
+    ports:
+      - 9000:9000
+      - 9001:9001
+    healthcheck:
+      test:
+        - CMD
+        - curl
+        - --silent
+        - --fail
+        - --output
+        - /dev/null
+        - http://localhost:9000/minio/health/live
+      start_period: 15s
+      interval: 5s
+      timeout: 10s
+      retries: 10
+    command:
+      - server
+      - /data
+      - --console-address
+      - :9001
+
+  mc:
+    image: minio/mc
+    container_name: mc
+    environment:
+      - AWS_ACCESS_KEY_ID=admin
+      - AWS_SECRET_ACCESS_KEY=password
+      - AWS_REGION=us-east-1
+    networks:
+      bufstream:
+    depends_on:
+      minio:
+        condition: service_healthy
+    entrypoint: >
+      /bin/sh -c "
+      until (/usr/bin/mc alias set minio http://minio:9000 admin password) do echo '...waiting...' && sleep 1; done;
+      /usr/bin/mc mb minio/bufstream;
+      /usr/bin/mc anonymous set public minio/bufstream;
+      tail -f /dev/null
+      "
+
+  bufstream:
+    image: bufbuild/bufstream:0.3.27
+    hostname: bufstream
+    container_name: bufstream
+    depends_on:
+      minio:
+        condition: service_healthy
+      postgres:
+        condition: service_healthy
+    environment:
+      - BUFSTREAM_KAFKA_HOST=bufstream
+      - BUFSTREAM_KAFKA_PUBLIC_HOST=bufstream
+    networks:
+      bufstream:
+    ports:
+      - 9089:9089
+      - 9092:9092
+    # Edit bufstream.yaml within this repository to change configuration.
+    volumes:
+      - ./bufstream.yaml:/bufstream.yaml
+    healthcheck:
+      test:
+        - CMD
+        - /usr/local/bin/bufstream
+        - admin
+        - status
+        - --exit-code
+        - --url
+        - http://127.0.0.1:9089
+      start_period: 15s
+      interval: 5s
+      timeout: 10s
+      retries: 10
+    command:
+      - serve
+      - --config
+      - /bufstream.yaml
+
+  akhq:
+    image: tchiotludo/akhq:0.25.0
+    container_name: akhq
+    stop_signal: SIGKILL
+    networks:
+      bufstream:
+    ports:
+      - 8080:8080
+    depends_on:
+      bufstream:
+        condition: service_healthy
+    healthcheck:
+      test:
+        - CMD
+        - curl
+        - --silent
+        - --fail
+        - --output
+        - /dev/null
+        - http://localhost:28081/health
+      start_period: 15s
+      interval: 5s
+      timeout: 10s
+      retries: 10
+    environment:
+      AKHQ_CONFIGURATION: |
+        akhq:
+          connections:
+            bufstream-local:
+              properties:
+                bootstrap.servers: "bufstream:9092"
+                client.id: "akhq;broker_count=1;host_override=bufstream"
+
+networks:
```

Comment:
?

Reply:
We can either place them all on the same network to ensure that they can communicate with each other, or I can toss them all on the host network. Using the host network has more chance for port conflicts with the current user.

Additionally, I wanted to be able to expand this to use multiple bufstreams to show that they'd share the metadata. However, we don't have yet the option to configure the admin port over environment variables - which means that if we're using host network and we want to have two bufstream deployments, we'd need to make two copies of the config file for bufstream. I wanted to avoid doing that if we expanded it to multiple bufstream pods.

In any case, removed it and put everything on the host network.

Reply:
Host network sounds right to me - I want users to be able to call this